src/components/cylc/log/Log.vue

```diff
@@ -16,15 +16,18 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
 <template>
-  <div>
-    <pre><span v-for="(log, index) in computedLogs" :key="index">{{log}}</span></pre>
-  </div>
+  <pre><span
+    v-for="(log, index) in computedLogs"
+    :key="index"
+    :class="wordWrap ? 'text-pre-wrap' : 'text-pre'"
+  >{{ log }}</span></pre>
```

Comment:
@oliver-sanders This is the only change to the code renderer, applying the `text-pre-wrap` or `text-pre` class

Reply:
(not suggesting we don't accept this, but saying we shouldn't go down this road unless it's the only option)

Reply:
Yeah don't worry, I didn't have any intention of doing so 😄